charts/newrelic-infra-operator/templates/_helpers.tpl

```diff
@@ -35,31 +11,7 @@ Create chart name and version as used by the chart label.
 Common app label
 */}}
 {{- define "newrelic-infra-operator.appLabel" -}}
-app.kubernetes.io/name: {{ include "newrelic-infra-operator.name" . }}
-{{- end -}}
-
-{{/*
-Common labels
-*/}}
-{{- define "newrelic-infra-operator.labels" -}}
-{{ include "newrelic-infra-operator.appLabel" . }}
-helm.sh/chart: {{ include "newrelic-infra-operator.chart" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
-{{- if .Chart.AppVersion }}
-app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-{{- end }}
-app.kubernetes.io/managed-by: {{ .Release.Service }}
-{{- end -}}
-
-{{/*
-Create the name of the service account to use
-*/}}
-{{- define "newrelic-infra-operator.serviceAccountName" -}}
-{{- if .Values.serviceAccount.create -}}
-    {{ default (include "newrelic-infra-operator.fullname" .) .Values.serviceAccount.name }}
-{{- else -}}
-    {{ default "default" .Values.serviceAccount.name }}
-{{- end -}}
+app.kubernetes.io/name: {{ include "newrelic.common.naming.name" . }}
```

Comment:
we could move this to the template since is just adding a line WDYT?

Reply:
Good catch 👍. Those are even already provided by common.labels.
Fixed on https://github.com/newrelic/newrelic-infra-operator/pull/160/commits/7901df87ff05411e3c14221e0edf1489f51f6c0d